src/views/admin/AdminView.tsx

```diff
@@ -23,20 +22,59 @@ export const AdminView = () => {
       window.location.reload()
     }
   }
+
+  const handleExportClick = () => {
+    const storageKeys = Object.keys(window.localStorage)
+    const storage = storageKeys.reduce((acc, key) => {
+      const rawValue = window.localStorage.getItem(key)
+      if (rawValue) {
+        acc[key] = JSON.parse(rawValue)
+      }
+
+      return acc
+    }, {} as Record<string, unknown>)
+    const jsonStorage = JSON.stringify(storage)
+
+    const linkElement = document.createElement('a')
+    linkElement.setAttribute('href', 'data:application/json;charset=utf-8,' + encodeURIComponent(jsonStorage))
+    linkElement.setAttribute('download', 'atlas-export.json')
+    linkElement.click()
+  }
+
+  const handleFileChange = async (e: Event) => {
+    const file = (e.target as HTMLInputElement).files?.[0]
+    if (!file) return
+    const fileText = await file.text()
+    const storage = JSON.parse(fileText)
+    Object.keys(storage).forEach((key) => {
+      window.localStorage.setItem(key, JSON.stringify(storage[key]))
+    })
+  }
+
+  const handleImportClick = async () => {
```

Comment:
Is it async?
Also, we could add some error handling here. For example, if you import damaged JSON, we could display information that imported JSON is not correct.